extensions/amp-analytics/0.1/vendors.js

```diff
@@ -107,6 +107,44 @@ export const ANALYTICS_CONFIG = {
       'xhrpost': false,
       'image': true
     }
+  },
+
+  'chartbeat': {
+    'vars': {
+      'decayTime': 15
+    },
+    'requests': {
+      'host': 'https://ping.chartbeat.net',
+      'interval': '${host}/ping?h=${domain}&' +
+        'p=${canonicalPath}&' +
+        'u=${clientId}&' +
+        'd=${canonicalHost}&' +
+        'g=${uid}&' +
+        'g0=${sections}&' +
+        'g1=${authors}&' +
+        'g2=${zone}&' +
+        'g3=${sponsorName}&' +
+        'g4=${contentType}&' +
+        'x=${scrollTop}&' +
+        'w=${screenHeight}&' +
+        'j=${decayTime}&' +
```

Comment:
> Not sure what decay time is.

Decay time is the amount of time in seconds till our system should should assume that the page session has ended. This allows us to release data through the system more reliably. In our data collection JS, there is a concept of "back-off" which causes data to be sent less frequently if the user is not engaged with the content. That causes the decay time to increase which causes our system to wait longer for the next request to be sent.

> Is this related to the timer trigger? Do you want to define the var there?

Yeah, that makes sense.